Makefile

```diff
@@ -126,6 +126,7 @@ smoketest:
 	tinygo build -size short -o test.elf -target=circuitplay-express examples/i2s
 	tinygo build -size short -o test.elf -target=gameboy-advance     examples/gba-display
 	tinygo build -size short -o test.elf -target=itsybitsy-m4        examples/blinky1
+	tinygo build -size short -o test.elf -target=stm32f103rb-nucleo  examples/blinky1
```

Comment:
I would name the target `nucleo-f103rb` so it matches the name on the datasheet.